ansible/roles/ocp4-workload-dil-agile-integration/tasks/remove_workload.yml

```diff
@@ -0,0 +1,142 @@
+---
+# Implement your Workload deployment tasks here
+- name: vars
+  set_fact:
+      operators_project: 'workshop-operators'
+
+- name: Uninstall Streams
+  block:
+    - name: Delete Subscription
+      k8s:
+        state: absent
+        api_version: operators.coreos.com/v1alpha1
+        kind: Subscription
+        name: amq-streams
+        namespace: '{{ operators_project }}'
+    - name: Collect CustomResourceDefintions
+      k8s_info:
+        api_version: apiextensions.k8s.io/v1beta1
+        kind: CustomResourceDefinition
+        label_selectors:
+        - app=strimzi
```

Comment:
Please use consistent list indentaton. If `block` is indented then `label_selectors` here should also be indented. Either use extra indentation of lists everywhere or nowhere.